src/Evolution/Initialization/Evolution.hpp

```diff
@@ -32,6 +32,28 @@ struct Inertial;
 }  // namespace Frame
 /// \endcond
 
+namespace Evolution_detail {
+// Global time stepping
+template <typename Metavariables,
+          Requires<not Metavariables::local_time_stepping> = nullptr>
```

Comment:
This isn't new code, but while this is all changing this can be changed from a Requires to an overload.

Reply:
Oops, never mind.  Can't partially specialize functions.  (And "overload" was the wrong word choice above, which I think increased the confusion.)